evergreen/template_ssdlc_compliance_report.md

```diff
@@ -43,15 +43,13 @@ Blocked on <https://jira.mongodb.org/browse/DRIVERS-2892>.
 
 The MongoDB SSDLC policy is available <a href="https://docs.google.com/document/d/1u0m4Kj2Ny30zU74KoEFCN4L6D_FbEYCaJ3CQdCYXTMc">here</a>.
 
-## Third-darty dependency information
+## Third-party dependency information
 
-There are no dependencies to report vulnerabilities of.
-Our [SBOM](https://docs.devprod.prod.corp.mongodb.com/mms/python/src/sbom/silkbomb/docs/CYCLONEDX/) lite
-is <https://github.com/mongodb/mongo-efcore-provider/blob/v${PACKAGE_VERSION}/sbom.json>.
+Our third party report is available <a href="https://us-west-2.console.aws.amazon.com/s3/object/csharp-driver-release-assets?region=us-west-2&bucketType=general&prefix=${PRODUCT_NAME}/${PACKAGE_VERSION}/augmented-sbom.json">here</a>.
```

Comment:
Intentionally using the same folder as the C# driver I assume?

Reply:
yes